src/hawkbit-client.c

```diff
@@ -544,6 +545,13 @@ gboolean install_complete_cb(gpointer ptr)
                 if (result->install_success) {
                         g_message("Software bundle installed successful.");
                         feedback(feedback_url, action_id, "Software bundle installed successful.", "success", "closed", NULL);
+                        if (hawkbit_config->post_update_reboot) {
+                                process_deployment_cleanup();
+                                sync();
+                                if(reboot(RB_AUTOBOOT) < 0){
+					g_warning("Reboot failed - reboot() call returned: -1");
+				}
```

Comment:
1. Use space instead of tabs to be consistent with the style of the existing code.
2. Add spacing around the `if`'s parentheses.
3. Instead of printing that `reboot()` may have returned `-1` in any case of failure, you should parse the value of `errno` here to give a more meaningful result.
4. IMHO if `reboot()` fails, it should be considered a critical failure and not just a warning, because the user clearly specified to reboot after an update but the program could not execute as expected. I am open to different opinions on this one though.

Reply:
@ctenruh-phytec For the style issues, you can easily have a look at the 'uncrustify' test result: https://github.com/rauc/rauc-hawkbit-updater/pull/39/checks?check_run_id=1777796976 You can also run 'uncrustify' manually from the repo by calling `./uncrustify.sh`.

Reply:
thanks for the review.